samples: maxim_ds3231: increase variable size in min_alarm_handler

CharlesDias opened the pull request:
Change the variable **us** from uint8_t to uint16_t within the min_alarm_handler function. This update allows the function to correctly handle and print values in the range of 0 to 999.

kartben commented:
While the fix looks good (thanks!) I am confused by the printk? It's printing "second.ns" so the millisecond part is lost, is it not?

kartben commented:
Ah, nevermind, it's the `us` variable name that confused me... Should be `ms` right?

CharlesDias commented:
Yes, you right! It would be better to change it to **ms**. Should I add a new commit to change its name, or just add in the previous one and force it?

kartben commented:
> Yes, you right! It would be better to change it to **ms**. Should I add a new commit to change its name, or just add in the previous one and force it?

Good question :) I think it makes sense to have everything squashed into one commit as part of the same "getting milliseconds right" effort. And then force push indeed. Thanks!

CharlesDias committed:
samples: maxim_ds3231: increase variable size in min_alarm_handler

Change the variable name from **us** to **ms** and increase its size
to **uint16_t** within the min_alarm_handler function. This update
allows the function to correctly handle the millisecond values in the
range of 0 to 999.

Signed-off-by: Charles Dias <charlesdias.cd@outlook.com>

CharlesDias commented:
Done. Thanks!
